pkg/restapi/vc/operation/operations.go

```diff
@@ -1537,7 +1537,7 @@ func getPresentationSigningOpts(opts *SignPresentationOptions) []crypto.SigningO
 }
 
 func (o *Operation) validateCredentialProof(vcByte []byte, opts *CredentialsVerificationOptions) error {
-	vc, err := o.parseAndVerifyVC(vcByte)
+	vc, err := o.parseAndVerifyVCStrictMode(vcByte)
```

Comment:
what about VPs

Reply:
Currently, no options available for VP - 

https://github.com/hyperledger/aries-framework-go/blob/7fc555ef6cb0aa37b5d4af0f43061eafc801fdc6/pkg/doc/verifiable/presentation.go#L312-L315
cc : @kdimak